ui/raidboss/emulator/data/network_log_converter/LineEvent0x00.ts

```diff
@@ -1,14 +1,24 @@
 import LineEvent from './LineEvent';
+import LogRepository from './LogRepository';
+
+export type LineEvent0x00Parts = [
+  string, // 0: ID
+  string, // 1: Timestamp
+  string, // 2: Type
+  string, // 3: Speaker
+  string, // 4: Message
+  string, // 5: Checksum
+];
 
 // Chat event
 export class LineEvent0x00 extends LineEvent {
-  constructor(repo, line, parts) {
+  constructor(repo: LogRepository, line: string, public parts: LineEvent0x00Parts) {
     super(repo, line, parts);
     if (this.parts.slice(4).join('|').split('\u001f\u001f').length > 1)
       this.invalid = true;
   }
 
-  convert() {
+  convert(_: LogRepository): void {
```

Comment:
Why is this signature different from the other `convert`s? Also, I would expect LineEvent to have this as an optional parameter?

Reply:
We had a whole conversation on this in the PR for the parent `LineEvent` class, here's a (hopefully working) link:
https://github.com/quisquous/cactbot/pull/2875#discussion_r636566724

All of the `convert` signatures should include a `LogRepository` argument, unless I missed one somewhere.

Reply:
I guess what I am trying to ask is why does 0x00 have this signature, which is different from all the other converts, aside from 0x1F (which is understandable)

Reply:
> All of the convert signatures should include a LogRepository argument, unless I missed one somewhere.

It looks like you missed a number of them.

Reply:
They should have all said they had a `LogRepository` param, apparently my regex wasn't correct for checking these. I'll fix for the next commit, thanks for spotting it.